packages/odyssey-storybook/src/components/odyssey-mui/MenuButton/MenuButton.stories.tsx

```diff
@@ -91,7 +91,7 @@ const storybookMeta: Meta<MenuButtonProps> = {
       description: "The <MenuItem> components within the Menu",
       table: {
         type: {
-          summary: "[MenuItem | Divider | ListSubheader]",
+          summary: "[MenuItem | Divider | ListSubheader | NullElement]",
```

Comment:
Isn't this documentation only?

Does it need to be in the props table?

Reply:
I'm not quite sure what you're asking here. Can you clarify?

Reply:
Props table is just a string. We don't need to put TypeScript types in here in our docs; TS handles that for you in the component.